### PR TITLE
Add total dividends to portfolio table

### PR DESCRIPTION
**Pull request by krazemon**

### CODE REVIEW

File: css/app.css [hunk 1]
```diff
@@ -671,7 +671,7 @@ html, body {
 }
 
 .anonymous {
-  float:right;
```
Comment (krazemon) [inline css/app.css]: Spacing issues I guess?

File: js/components/PortfolioHeader.jsx [hunk 2]
```diff
@@ -10,14 +10,28 @@ var PortfolioHeader = React.createClass({
...
-        <span className="reps-available-info glyphicon glyphicon-info-sign" data-toggle="popover" data-placement="right" data-content={strings.REPS_AVAILABLE_INFO_CONTENT}></span></p>;
+      repsAvailable = (
+        <div>{strings.PORTFOLIO_HEADER_REPS(this.props.reps)}
+        <span className="reps-available-info glyphicon glyphicon-info-sign"
+          data-toggle="popover" data-placement="right"
+          data-content={strings.REPS_AVAILABLE_INFO_CONTENT}></span></div>
+      )
+    }
+
+    if (this.props.dividends) {
```
Comment (ritmatter) [inline js/components/PortfolioHeader.jsx]: Is there ever a case where there would not be a this.props.dividends?

**Comment by ritmatter** [comment]
LGTM just look at my question. Mostly interested out of curiosity

File: js/components/PortfolioHeader.jsx (hunk 2 above)
Comment (krazemon) [inline js/components/PortfolioHeader.jsx]: If the user had no investments, I'm pretty sure there would be no dividends.

Comment (krazemon) [inline js/components/PortfolioHeader.jsx]: I guess they'd just be 0 still though.

**Comment by krazemon** [comment]
Question replied to. Take a look and merge whenever.

File: js/components/PortfolioHeader.jsx (hunk 2 above)
Comment (ritmatter) [inline js/components/PortfolioHeader.jsx]: If a user had no investments, then they wouldn't be an investor for the category. But, the check isn't hurting anybody.

**Pull request merged**